.github/workflows/CD.yml

```diff
@@ -0,0 +1,32 @@
+name: Continuous Deployment
+
+# Only deploy app when branch is merged into main
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  buildAndDeploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v2
+
+    - name: Setup Node
+      uses: actions/setup-node@v2
+      with:
+      node-version: '14'
+    
+    - name: Download dependencies
+      run: npm install
+    
+    - name: Build deployable
+      run: npm run build
+    
+    - name: Sync to S3
+      run: aws s3 sync --delete build s3://$S3_BUCKET
+      env:
+        S3_BUCKET: ${{ secrets.S3_BUCKET }}
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_KEY }}
+        AWS_DEFAULT_REGION: 'us-east-2'
```

Comment:
Don't know if this'll work... works locally but no easy way to test without just going for it.